hazelcast-client/src/main/java/com/hazelcast/client/config/ClientClasspathXmlConfig.java

```diff
@@ -56,19 +56,6 @@ public ClientClasspathXmlConfig(String resource, Properties properties) {
     }
 
     /**
-     * Creates a config which is loaded from a classpath resource. The System.properties are used to
-     * resolve variables in the XML.
-     *
-     * @param classLoader the ClassLoader used to load the resource
-     * @param resource    the resource, an XML configuration file from the classpath
-     * @throws IllegalArgumentException if classLoader or resource is {@code null}, or if the resource is not found
-     * @throws com.hazelcast.core.HazelcastException if the XML content is invalid
-     */
-    public ClientClasspathXmlConfig(ClassLoader classLoader, String resource) {
```

Comment:
This is public API. And that means we can't delete a method here.

Reply:
@sancar  We put it new at 3.9 and it is not published yet. Hence, it should not be a problem, right?